### PR TITLE
Improve website

### PR DESCRIPTION
**Pull request by ademariag**
Improve website and documention for kapitan.dev

* Adds tons of documentation
* Simplify the website experience
* Simpler landing page on GitHub, with link to website

# TODO:
* Still lots of documentation is obsolete and needs updating
* Add new demo of kapitan

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,66 +8,19 @@
...
-* **Mailing List** [kapitan-discuss@googlegroups.com](mailto:kapitan-discuss@googlegroups.com)
+Join the community [`#kapitan`](https://kubernetes.slack.com/archives/C981W2HD3)
 
-How is it different from [`helm`](https://github.com/kubernetes/helm) and [`kustomize`](https://github.com/kubernetes-sigs/kustomize)? Please look at our [FAQ](https://kapitan.dev/#faq)!
+## [**Official site**](https://kapitan.dev) https://kapitan.dev
 
-<img src="docs/images/kapitan_logo.png" width="250">
 
-## Key Concepts
+## [**Quick Start**](https://kapitan.devkapitan_overview/#quickstart)
```
Comment (ramaro) [inline README.md:21]: link is wrong

File: README.md [hunk 2]
```diff
@@ -79,16 +32,16 @@ On Linux you can add `-u $(id -u)` to `docker run` to preserve file permissions.
 
 ### Pip
 
-Kapitan needs Python 3.7.
+Kapitan needs Python 3.6.
 
-**Install Python 3.7:**
+#### Install Python 3.6
```
Comment (ramaro) [inline README.md:37]: 3.7 is minimum, soon to be 3.8

File: README.md [hunk 3]
```diff
@@ -79,16 +32,16 @@ On Linux you can add `-u $(id -u)` to `docker run` to preserve file permissions.
...
 
-* Linux: `sudo apt-get update && sudo apt-get install -y python3.7-dev python3-pip python3-yaml`
+* Linux: `sudo apt-get update && sudo apt-get install -y python3.6-dev python3-pip python3-yaml`
```
Comment (ramaro) [inline README.md:39]: and here

File: docs/FAQ.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+---
+comments: true
+---
+# :kapitan-logo: FAQ
+## Why do I need Kapitan?
+
+With Kapitan, we worked to de-compose several problems that most of the other solutions are treating as one.
+
+1) ***Kubernetes manifests***: We like the jsonnet approach of using json as the working language. Jsonnet allows us to use inheritance and composition, and hide complexity at higher levels.
```
Comment (ramaro) [inline docs/FAQ.md:9]: Just jsonnet?

File: docs/README.md [hunk 5]
```diff
@@ -1,217 +1,86 @@
-# Kapitan: Generic templated configuration management for Kubernetes, Terraform and other things
+# :kapitan-logo: Kapitan: advanced configuration management tool
 
-![Unit Tests](https://github.com/kapicorp/kapitan/actions/workflows/test.yml/badge.svg)
```
Comment (ramaro) [inline docs/README.md:3]: We should keep these badges should still be at the top

File: docs/README.md [hunk 6]
```diff
@@ -1,217 +1,86 @@
...
+![downloads](https://img.shields.io/pypi/dm/kapitan)
 
-On Linux you can add `-u $(id -u)` to `docker run` to preserve file permissions.
+##### User
 
-#### Pip
+=== "Linux"
 
-Kapitan needs Python 3.6.
+    `$HOME/.local/lib/python3.6/bin`
```
Comment (ramaro) [inline docs/README.md:62]: 3.7

**Comment by ademariag** [comment]
See note: 
"Still lots of documentation is obsolete and needs updating"

will do it another PR

File: docs/FAQ.md (hunk 4 above)
Comment (ademariag) [reply docs/FAQ.md:9]: Still lots of documentation is obsolete and needs updating

File: docs/README.md (hunk 5 above)
Comment (ademariag) [reply docs/README.md:3]: ok for the github page, but even for the website? Looks ugly. That's more a github thing?

Comment (ademariag) [reply docs/README.md:3]: I have left it as it is in the github repo README, but on the website I have moved it to more relevant places. I don't think it belongs to the website

Comment (ramaro) [reply docs/README.md:3]: got it

**Pull request merged**